samples/mqtt/basic_pub_sub/main.cpp

```diff
@@ -42,7 +46,19 @@ static void s_printHelp()
         "websockets)\n");
     fprintf(stdout, "proxy_host: if you want to use a proxy with websockets, specify the host here (optional).\n");
     fprintf(
-        stdout, "proxy_port: defaults to 8080 is proxy_host is set. Set this to any value you'd like (optional).\n\n");
+        stdout, "proxy_port: defaults to 8080 is proxy_host is set. Set this to any value you'd like (optional).\n");
+
+    fprintf(stdout, "  x509: Use the x509 credentials provider while using websockets (optional)\n");
+    fprintf(stdout, "  x509rolealias: Role alias to use with the x509 credentials provider (required for x509)\n");
```

Comment:
consistency: The existing flags use underscores, but the new flags are allonelongword